From ea92cca7c1d4dfbd6a563588a6ea9b56a764fc39 Mon Sep 17 00:00:00 2001
From: lujingxiao <lujingxiao@huawei.com>
Date: Wed, 23 Jan 2019 15:31:56 +0800
Subject: [PATCH 12/27] bump: bump containerd to 1.2.0.2

reason: bump containerd to 1.2.0.2 after cherry-picked patches from
upstream:
- runtime:  Add timeout and cancel to shim fifo open
- runtime:  fix pipe in broken may cause shim lock forever for runtime v1
- runtime:  fix pipe in broken may cause shim lock forever for runtime v2
- runtime:  Use named pipes for shim logs
- shim:  Increase reaper buffer size and non-blocking send
- shim:  optimize shim lock in runtime v1

Change-Id: Ibd7574e2ab18a2f783c694931101e1459bc779ad
Signed-off-by: lujingxiao <lujingxiao@huawei.com>
---
 hack/containerd.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hack/containerd.spec b/hack/containerd.spec
index f53c37b..c7d358d 100644
--- a/hack/containerd.spec
+++ b/hack/containerd.spec
@@ -3,7 +3,7 @@
 Version:        1.2.0
 
 Name:           containerd
-Release:        1%{?dist}
+Release:        2%{?dist}
 Summary:        An industry-standard container runtime
 License:        ASL 2.0
 URL:            https://containerd.io
-- 
2.7.4.3

